README.md

```diff
@@ -1 +1,60 @@
-# Capstone1
\ No newline at end of file
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# wsc2022
+
+>World Snooker Championship capstone project of module 1
+
+
+## Built With
+
+- HTML & CSS
+- JavaScript
+
+## Live Demo
+-Live demo link: (https://app-developer8866.github.io/wsc2022.github.io/)
+
+## Video Presentation
+-Video [presentrtion](https://www.loom.com/share/ea2dbf75e5964a2c8bdccec9b1c98b58)
+
+
+## Getting Started
+
+**If you want to this project kindly check the [ GitHub repository](https://github.com/App-Developer8866/Capstone_1)**
```

Comment:
- [ ] Kindly use the new Readme template which is the new standard and requirement for all projects from here on.[Link](https://github.com/microverseinc/readme-template/blob/master/README.md)